README.md

```diff
@@ -53,9 +53,7 @@ ArcticDB is designed from the outset to be resilient; there is no single point o
 | --------------------- | - | - |
 | S3                 | ✔️ | ✔️ |
 | LMDB               | ✔️ | ✔️ |
-| Azure Blob Storage | ✔️ | ➖ |
-
-Support for Azure Blob Storage in conda-forge is tracked in [#519](https://github.com/man-group/ArcticDB/issues/519).
+| Azure Blob Storage | ✔️ | ✔️ |
```

Comment:
This will be visible a bit early.  People won't see it till we get a release out.

Reply:
Would adapting the comment to mention that the support will be introduced in 3.1.0 (or even in 3.0.0) make sense?